bootstrap/knownhosts.go

```diff
@@ -124,6 +124,14 @@ func newGittableURL(ref string) (*url.URL, error) {
 	return url.Parse(ref)
 }
 
+// Clean up the SSH host and remove any key identifiers. See:
+// git@github.com-custom-identifier:foo/bar.git
+// https://buildkite.com/docs/agent/ssh-keys#creating-multiple-ssh-keys
+var repoSSHKeySwitcherRegex = regexp.MustCompile(`-[a-z0-9\-]+$`)
+func repoSSHKeySwitcherStrip(host string) (string) {
```

Comment:
Wouldn't this strip these values from legit hostnames?

Reply:
Perhaps `stripAliasesFromGitHost` might be clearer?

Reply:
I don't mind what it's called, just extracted it out the way :-)